### PR TITLE
Bug fix

### PR DESCRIPTION
**Pull request by animeshgarai09**
Done some refactoring and changed few CSS styles.

### CODE REVIEW

**Review by Dev-AkshatGupta — APPROVED** [review]
This is wholly awesome . :) So much amazing CSS properties

File: components/badge/badge.scss [hunk 1]
```diff
@@ -52,18 +52,18 @@
     .badge {
         display: inline-flex;
         align-items: center;
-        justify-content: center;
-        border-radius: 10px;
+        line-height: 0;
+        border-radius: rem(10);
         color: #ffffff;
```
Comment (vrinda-mahajan) [inline components/badge/badge.scss:57]: You can reuse your var here.

**Review by vrinda-mahajan — APPROVED** [review]
Everything else looks good to me.

**Comment by yash9263** [comment]
This looks amazing. Great work

File: components/badge/badge.scss (hunk 1 above)
Comment (animeshgarai09) [reply components/badge/badge.scss:57]: oh sure. thanks

**Review by yash9263 — APPROVED** [review]

**Pull request merged**